lib/far_mar/market.rb

```diff
@@ -0,0 +1,88 @@
+require 'csv'
+
+class FarMar::Market < FarMar::Base
+
+  MARKET_ARRAY = CSV.read("support/markets.csv")
+
+  attr_reader :name,
+              :address,
+              :city,
+              :county,
+              :state,
+              :zip
+
+    def initialize(attrs)
+      super(attrs[:id])           # @id = attrs[0].to_i
+      @name    = attrs[:name]     # @name = attrs[1]
```

Comment:
Just a comment on your comment over there, if you used `@name = attrs[1]`, (i.e. using an array rather than a hash) then you could easily have the `.all` method `FarMar::Base`